src/audio/volume/volume_hifi3.c

```diff
@@ -44,6 +65,7 @@ static void vol_s24_to_s24_s32(struct comp_dev *dev, struct audio_stream *sink,
 			       const struct audio_stream *source,
 			       uint32_t frames)
 {
+#if (OPT == OPT0)
```

Comment:
Can we have a new function for each optimization type.

Reply:
Could we just remove all but OPT2 code version?